metabill/src/metabill/core.clj

```diff
@@ -24,8 +25,38 @@
     (spit f (pr-str d))
     d))
 
+(defn- print-err [& msg]
+  (println msg)
```

Comment:
Is this `println` intended? It would duplicate the same message to both stdout and stderr.